Dockerfile

```diff
@@ -92,6 +92,20 @@ RUN wget https://github.com/samtools/htslib/releases/download/1.9/htslib-1.9.tar
 #### R packages
 ###############
 
+# Install BiocManager and the desired version of Bioconductor
+RUN R -e "install.packages('BiocManager', dependencies=TRUE, repos='https://bioconductor.org/packages/3.16/bioc')"
+RUN R -e "BiocManager::install(version = '3.16', repos = 'https://bioconductor.org/packages/3.16/bioc')"
+
+# These packages are for the genomic region analysis for snv-callers
+RUN R -e 'BiocManager::install(c("annotatr", "TxDb.Hsapiens.UCSC.hg38.knownGene", "org.Hs.eg.db", "BSgenome.Hsapiens.UCSC.hg19", "BSgenome.Hsapiens.UCSC.hg38"))'
```

Comment:
@zzgeng if this passes, can you update this dockerfile to put all of the R packages we are using from bioc in this command - ideally in alpha order 😬 - such to ensure they are being installed using the 3.16 version of bioc